static/data/data.json

```diff
@@ -150,7 +150,9 @@
       "emoji": "🐦",
       "tags": ["rust", "tools"],
       "description": "Simple instrumentation profiler for Rust.",
-      "featured": true      
+      "featured": true,
+      "extendedDescription": "The friendly little instrumentation-based CPU profiler for Rust",
+      "featureImage": "https://github.com/EmbarkStudios/puffin/blob/main/puffin.jpg?raw=true        "      
```

Comment:
Those spaces at the end of the `featureImage` URL ;)

Reply:
oh weird, does still work though but will fix

Reply:
there